cmd/fyne/commands/build.go

```diff
@@ -20,18 +20,23 @@ func (b *builder) build() error {
 		goos = targetOS()
 	}
 
-	args := []string{}
+	args := []string{"build"}
+	env := os.Environ()
+	env = append(env, "CGO_ENABLED=1") // in case someone is trying to cross-compile...
+
 	if goos == "windows" {
 		if b.release {
-			args = append(args, "build", "-ldflags", "-s -w -H=windowsgui")
+			args = append(args, "-ldflags", "-s -w -H=windowsgui")
 		} else {
-			args = append(args, "build", "-ldflags", "-H=windowsgui")
+			args = append(args, "-ldflags", "-H=windowsgui")
 		}
 	} else {
+		if goos == "darwin" {
+			env = append(env, "CGO_CFLAGS=-mmacosx-version-min=10.11")
```

Comment:
Should we perhaps move out the minimum version to a string contestant at the top of the file? I imagine something like `const minimumSupportedOSX = “10.11”?

Reply:
I wasn't sure. It also appears in the release packaging file, so a single const just here might imply something it shouldn't? (i.e. that changing just once changes all)

Reply:
Good point. We can change it in the future in case it seems relevant then.

Reply:
Yes. We could consider it being a configuration - but we don't currently do things at that level yet.
Maybe once we have an (optional) metadata file we could do more...